_events/2023-01-25-htcondor-osg.md

```diff
@@ -1,11 +1,11 @@
 ---
-short_title: Joint OSG All Hands Meeting and HTCondor Week July 10-14
-title: "Save the Dates for the Joint HTCondor Week - OSG All-Hands Meeting - 10-14 July 2023"
+short_title: Announcing Throughput Computing 2023 to be held July 10-14
+title: "Save the Dates for Throughput Computing 2023 - a joint HTCondor OSG All-Hands event"
```

Comment:
“a joint HTCondor OSG All-Hands event” doesn’t make sense to me. I would say it is either “a joint HTCondor/OSG event” or spell out the whole thing as “a joint HTCondor Week / OSG All-Hands Meeting event”. Given that we say the latter enough and this is a title, I would vote for the shorter version.

Reply:
I went with the short version

Reply:
I used the shorter version